### PR TITLE
Update Preboarding_and_Onboarding.rst

### PR DESCRIPTION
**Pull request by VenkyOS**
Signed-off-by: Venkataraman Padmanabhan <venkataraman.padmanabhan@seagate.com>

### CODE REVIEW

File: doc/Preboarding_and_Onboarding.rst [hunk 1]
```diff
@@ -51,6 +51,14 @@ The preboarding process is the first process that must be completed after config
 Onboarding
 ===========
 
+If the Elasticsearch service is not running, execute the below mentioned commands.
+
```
Comment (mukul-seagate11) [inline doc/Preboarding_and_Onboarding.rst:55]: Its a known reported Elasticsearch issue which can be avoided by executing mentioned commands

File: doc/Preboarding_and_Onboarding.rst [hunk 2]
```diff
@@ -51,6 +51,14 @@ The preboarding process is the first process that must be completed after config
...
+::
+
+ sed -i '11s/host:/host: 127.0.0.1/' /etc/csm/database.yaml
+
```
Comment (mukul-seagate11) [inline doc/Preboarding_and_Onboarding.rst:59]: remove space

File: doc/Preboarding_and_Onboarding.rst (hunk 1 above)
Comment (VenkyOS) [reply doc/Preboarding_and_Onboarding.rst:55]: Resolved.

File: doc/Preboarding_and_Onboarding.rst (hunk 2 above)
Comment (VenkyOS) [reply doc/Preboarding_and_Onboarding.rst:59]: Space is added due to formatting.

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Pull request merged**